definitions.yml

```
@@ -26,3 +26,22 @@ definitions:
         type: string
       pec:
         type: string
+  Comune:
```

we use english for the attributes

```suggestion
  Municipality:
```